tests/manual_test_mcd.py

```diff
@@ -0,0 +1,61 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2019 EdNoepel
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import sys
+from web3 import Web3, HTTPProvider
+
+from pymaker import Address
+from pymaker.deployment import DssDeployment
+from pymaker.keys import register_keys
+from pymaker.numeric import Wad
+
+
+web3 = Web3(HTTPProvider(endpoint_uri="https://parity0.kovan.makerfoundation.com:8545",
```

Comment:
pls check comments within Readme file re local node, kovan-addresses.json and the way to retrieve collateral